From 2d73dcf8869edc9ddc391d70cfbf0bef84d20ac1 Mon Sep 17 00:00:00 2001
From: Andy Grover <agrover@redhat.com>
Date: Wed, 30 Jul 2014 17:20:34 -0400
Subject: [redhat] RHMAINTAINERS: Add agrover for drivers/target

Message-id: <1406740834-20017-1-git-send-email-agrover@redhat.com>
Patchwork-id: 86847
O-Subject: [RHEL7 PATCH] RHMAINTAINERS: Add myself for drivers/target
Bugzilla:
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Signed-off-by: Andy Grover <agrover@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/RHMAINTAINERS b/redhat/RHMAINTAINERS
index 302c06e..b784b80 100644
--- a/redhat/RHMAINTAINERS
+++ b/redhat/RHMAINTAINERS
@@ -581,6 +581,12 @@ M: Doug Ledford <dledford@redhat.com>
 S: Maintained
 F: drivers/scsi/
 
+SCSI TARGET (LIO)
+M: Andy Grover <agrover@redhat.com>
+S: Maintained
+F: drivers/target/
+F: include/target/
+
 SCTP PROTOCOL
 M: Daniel Borkmann <dborkman@redhat.com>
 M: Neil Horman <nhorman@redhat.com>
-- 
1.7.1